draft-ietf-quic-transport.md

```diff
@@ -595,11 +592,9 @@ It carries cryptographic handshake messages and acknowledgments.  It is used
 by a server that wishes to perform a stateless retry (see
 {{stateless-retry}}).
 
-The connection ID field in a Server Stateless Retry packet contains a server
-selected value, see {{connection-id}}.
-
-The packet number field echoes the packet number of the triggering client
-packet.  This allows a client to verify that the server received its packet.
+The packet number and connection ID fields echo the packet number of the
```

Comment:
This says the connection ID field contain's the original packet's packet number.  Don't think that's correct.